src/main/java/frc/robot/commands/MovePlacerToNextBar.java

```diff
@@ -11,15 +11,15 @@
 
     private static final RootNamespace rootNamespace = new RootNamespace("move placers to next bar");
 
-    public static final Supplier<Double> GET_OVER_STALL_TIMEOUT =
-            rootNamespace.addConstantDouble("get over stall timeout", 0.2);
+    public static final Supplier<Double> START_TIMEOUT =
+            rootNamespace.addConstantDouble("start timeout", 0.2);
 
     public MovePlacerToNextBar(ClimberPlacer placer) {
         super(
                 new MoveGenericSubsystem(placer, placer.RAISE_SPEED) {
```

Comment:
why?

Reply:
in the start of the encoder's velocity will be in the tolerance, we want to avoid this to be recognized as the end to the command.